app/models/price_group.rb

```diff
@@ -40,6 +40,10 @@ def global?
     facility.nil?
   end
 
+  def can_manage_price_group_members?
+    is_not_global || SettingsHelper.feature_on?(:can_manage_global_price_groups)
```

Comment:
can we use `is_not_global?` with the question mark?

Reply:
Not unless we change the method name; it'll hit an undefined method error.

I was assuming there was some reason relating to the `before_destroy` hook above that `is_not_global` doesn't have a question mark already--does `?` play oddly with symbols?